From: Arvind Sankar <nivedita@alum.mit.edu>
Date: Tue, 24 Dec 2019 16:10:02 +0100
Subject: efi/gop: Remove unused typedef
Patch-mainline: v5.6-rc1
Git-commit: 6c895c2fca8a8d4e740b5498b48f81111569502a
References: jsc#SLE-16407

We have stopped using gop->query_mode(), so remove the unused typedef
for the function prototype.

Signed-off-by: Arvind Sankar <nivedita@alum.mit.edu>
Signed-off-by: Ard Biesheuvel <ardb@kernel.org>
Cc: Borislav Petkov <bp@alien8.de>
Cc: James Morse <james.morse@arm.com>
Cc: Matt Fleming <matt@codeblueprint.co.uk>
Cc: Thomas Gleixner <tglx@linutronix.de>
Cc: linux-efi@vger.kernel.org
Link: https://lkml.kernel.org/r/20191224151025.32482-3-ardb@kernel.org
Signed-off-by: Ingo Molnar <mingo@kernel.org>
Acked-by: Lee, Chun-Yi <jlee@suse.com>
---
 include/linux/efi.h |    4 ----
 1 file changed, 4 deletions(-)

--- a/include/linux/efi.h
+++ b/include/linux/efi.h
@@ -1480,10 +1480,6 @@ struct efi_graphics_output_protocol {
 	struct efi_graphics_output_protocol_mode *mode;
 };
 
-typedef efi_status_t (*efi_graphics_output_protocol_query_mode)(
-	struct efi_graphics_output_protocol *, u32, unsigned long *,
-	struct efi_graphics_output_mode_info **);
-
 extern struct list_head efivar_sysfs_list;
 
 static inline void
